opencti-platform/opencti-graphql/tests/02-integration/01-database/stix-filtering-test.ts

```diff
@@ -0,0 +1,104 @@
+import { describe, expect, it } from 'vitest';
+
+import { ADMIN_USER, buildStandardUser, testContext } from '../../utils/testQuery';
+import { isStixMatchFilterGroup } from '../../../src/utils/stix-filtering/stix-filtering';
+import type { FilterGroup } from '../../../src/utils/stix-filtering/filter-group';
+
+import stixReports from '../../data/stream-events/stream-event-stix2-reports.json';
```

Comment:
I could not rely on the data file `DATA-TEST-STIX2_v2.json` as it's exported data and not what we would find in stream events (that contains more data as it seems).

I've added a whole set of stix objects in tests/data/stream-events on which I base my tests.